### PR TITLE
Add warning when starting ephemeral workspace

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Anna Shumilova <ashumilo@redhat.com>

### What does this PR do?
Shows warning to user when starting ephemeral workspace.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12565

screenshot from 2019-02-28 13-47-16

### CODE REVIEW

**Review by akurinnoy — APPROVED** [review]

**Comment by ashumilova** [comment]
ci-test

File: dashboard/src/components/widget/notification/che-warning-notification.directive.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2015-2018 Red Hat, Inc.
```
Comment (olexii4) [inline dashboard/src/components/widget/notification/che-warning-notification.directive.ts:2]: 2019

**Review by olexii4 — APPROVED** [review]

File: dashboard/src/components/widget/notification/che-warning-notification.directive.ts (hunk 1 above)
Comment (ashumilova) [reply dashboard/src/components/widget/notification/che-warning-notification.directive.ts:2]: first should be changed here https://github.com/eclipse/che-parent/blob/master/pom.xml#L84 
to pass validation

**Comment by bmicklea** [comment]
The message here is perfect - but shouldn't there be a similar (persistent) warning in the IDE as well?

**Pull request merged**